ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -190,36 +190,44 @@ TFCore::getTensorTypeFromTF (DataType tfType)
  * @param tType	: the defined type of NNStreamer
  * @return the enum of defined tensorflow::DataType
```

Comment:
let's change the description (param and return)

Reply:
it's applied :)